docs/demos/lcore/coderabbit.md

```
@@ -0,0 +1,530 @@
+</g>
+</svg>
+
+---
+
+## Go specific
+
+* golangci-lint integration
+* Sandboxed
+* JSON output
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)